README.md

```diff
@@ -2,4 +2,50 @@
 
 **This is not an official Google product.**
 
-Privacy data mapping and related plans management.
+Pmap is a service for privacy data mapping and privacy plans management to
+conform with Alphabet privacy policies on data classification and inventory.
+
+## Use case
+
+Pmap can be used for users to map specific resources with wipeout plan to adopt
+or maintain privacy policies on data classification and inventory.
+
+## Architecture
+
+![pmap architecture](./docs/assets/pmap-arch.png)
+
+Pmap service will be triggered when snapshots as yaml files are uploaded to GCS
+bucket. A pubsub topic will subscribe the cloud storage notifications, and push
+the notification to cloud run service. Cloud run service will validate and
+enrich the annotations, and write the result to bigquery table.
+
+## End User workflow
+
+There are two ways for end users to use pmap service.
+
+1. Upload the resource annotation as a yaml file through a PR, when the PR is
+   merged, a github workflow will be triggered to snapshot the newly added files
+   and upload it to Pmap service's GCS bucket. User will be able to query the
+   result in bigquery table.
+2. A corn job will snapshot all yaml files and upload them to pmap service's GCS
+   bucket. Users will be able to query te result in bigquey table.
+
+## Installation
```

Comment:
As I've mentioned, I will handle the user facing part including user workflow and set up, you can see https://github.com/abcxyz/pmap/pull/178

Reply:
Updated.